qemu/tests/cfg/hotplug_mem_stress_ng.cfg

```diff
@@ -1,8 +1,10 @@
 - hotplug_mem_stress_ng:
     type = hotplug_mem_stress_ng
     mem_fixed = 4096
+    iomix_bytes = 2G
```

Comment:
May I ask why we use different iomix_bytes across different archs? Does  the free fs space matter across different arch? BTW, could you please paste the link that shows the latest version of stress-ng doesn't support iomix_bytes with percentage values? Thanks.

Reply:
@zhencliu For the stress-ng version question only,  I checked the repo during the review before, i think it is this one:
https://github.com/ColinIanKing/stress-ng/commit/7a59a6995b90cc8b737af35dc0ed707bee39216c

Reply:
@zhencliu about using different sizes depending on the archs is because we want to use fixed values, so having different archs with different memory assigned to the VM made me to did that way.
For the unsupport version question I think it could be the commit @yanan-fu has pointed, anyways to double confirm it I opened an issue in the repo so we can get an official answer: https://github.com/ColinIanKing/stress-ng/issues/559